.gitignore

```diff
@@ -1,3 +1,4 @@
 composer.lock
 phpunit.xml
 vendor
+.idea/
```

Comment:
That's the kind of feedback I was hoping for, instead of just a thumbs down. Bit more constructive - thanks.